### PR TITLE
Portable mode + async GUI version 2

### PR DESCRIPTION
**Pull request by K900**
Now with 100% less .\* imports ;)

### CODE REVIEW

**Comment by stephenwilliams** [comment]
All pulls in the launcher will have to wait til I get home from work

File: src/main/java/org/spoutcraft/launcher/SettingsHandler.java [hunk 1]
```diff
@@ -154,15 +155,14 @@ public class SettingsHandler {
...
-				} catch (Exception e) {
+                    input.close();
+				} catch (Exception ignored) {
 				}
 				try {
 					if (output != null)
 						output.close();
 				}
-				catch (Exception e)
+				catch (Exception ignored)
```
Comment (stephenwilliams) [inline src/main/java/org/spoutcraft/launcher/SettingsHandler.java:165]: No reason to change this

File: src/main/java/org/spoutcraft/launcher/SettingsHandler.java [hunk 2]
```diff
@@ -154,15 +155,14 @@ public class SettingsHandler {
 				e.printStackTrace();
 			} finally {
 				try {
-					if (input != null)
-						input.close();
```
Comment (stephenwilliams) [inline src/main/java/org/spoutcraft/launcher/SettingsHandler.java:158]: Keep ifs on one line

File: src/main/java/org/spoutcraft/launcher/SettingsHandler.java [hunk 3]
```diff
@@ -105,10 +106,11 @@ public class SettingsHandler {
 			FileOutputStream output = null;
 			try
 			{
-				out.getParentFile().mkdirs();
+                //noinspection ResultOfMethodCallIgnored
+                out.getParentFile().mkdirs();
```
Comment (stephenwilliams) [inline src/main/java/org/spoutcraft/launcher/SettingsHandler.java:110]: Why the indent, your formatting doesn't match the ones we use

**Comment by stephenwilliams** [comment]
@K900 I have decided to pull this request, but please do not go through and change all of our code next time if it is not part of the reason for you pull request. In this case your pull request was for Async Downloading and Portable mod, but your pull also included a lot of "fixes" that were not needed and your personal preference. 

Also make sure to follow the formatting standards that are set in the Spout plugins readme

File: src/main/java/org/spoutcraft/launcher/SettingsHandler.java (hunk 3 above)
Comment (K900) [inline src/main/java/org/spoutcraft/launcher/SettingsHandler.java:110]: Sorry, probably just pressed Tab

**Pull request merged**